### ISSUE TITLE
Podman rootless container with a custom network succeeds or fails based on which home directory the user has

### ISSUE THREAD

**Issue by Gchbg** [containers/podman#22227]
### Issue Description

I have encountered a bizarre issue running rootless containers within a user instance of systemd. When I attempt to create a network and run a container inside that network, podman succeeds or fails depending on the home directory of the user.

### Steps to reproduce the issue

Start with a pristine stock Debian trixie VM.

Prepare the system:
```
sudo sysctl -w net.ipv4.ping_group_range="0 4294967293"
sudo apt install --no-install-recommends podman uidmap netavark aardvark-dns iptables slirp4netns
```

Create two identical users. The only difference is that `yc` lives under `/home` and `nc` lives under `/var/lib`:
```
sudo groupadd -r -g 201 yc
sudo useradd -r -s /usr/sbin/nologin -l -d /home/yc -M -g yc -u 201 yc
sudo mkdir -m 770 /home/yc
sudo chown -R yc:yc /home/yc
printf 'yc:231072:65536\n' | sudo tee -a /etc/subuid
printf 'yc:231072:65536\n' | sudo tee -a /etc/subgid
sudo loginctl enable-linger yc

sudo groupadd -r -g 202 nc
sudo useradd -r -s /usr/sbin/nologin -l -d /var/lib/nc -M -g nc -u 202 nc
sudo mkdir -m 770 /var/lib/nc
sudo chown -R nc:nc /var/lib/nc
printf 'nc:296608:65536\n' | sudo tee -a /etc/subuid
printf 'nc:296608:65536\n' | sudo tee -a /etc/subgid
sudo loginctl enable-linger nc
```

Switch to `yc`, create a network, start a container:
```
sudo su -Pl yc -s /bin/sh -c 'XDG_RUNTIME_DIR="/run/user/$(id -u)" DBUS_SESSION_BUS_ADDRESS="unix:path=${XDG_RUNTIME_DIR}/bus" export XDG_RUNTIME_DIR DBUS_SESSION_BUS_ADDRESS && bash'
$ podman network create mynetyc
mynetyc
$ podman network inspect mynetyc
[
     {
          "name": "mynetyc",
          "id": "88439a2ba7d8e2fb3a3e46f314c2cc1582bd5aabbbaeb3bec3ab1855bc1ac21e",
          "driver": "bridge",
          "network_interface": "podman1",
          "created": "2024-03-31T22:59:32.217311329+03:00",
          "subnets": [
               {
                    "subnet": "10.89.0.0/24",
                    "gateway": "10.89.0.1"
               }
          ],
          "ipv6_enabled": false,
          "internal": false,
          "dns_enabled": true,
          "ipam_options": {
               "driver": "host-local"
          }
     }
]
$ podman run --rm -it --net=mynetyc alpine ping www.google.com
Resolved "alpine" as an alias (/etc/containers/registries.conf.d/shortnames.conf)
Trying to pull docker.io/library/alpine:latest...
Getting image source signatures
Copying blob bca4290a9639 done   |
Copying config ace17d5d88 done   |
Writing manifest to image destination
PING www.google.com (216.58.212.132): 56 data bytes
64 bytes from 216.58.212.132: seq=0 ttl=42 time=14.068 ms
64 bytes from 216.58.212.132: seq=1 ttl=42 time=19.104 ms
^C
```

Exit, switch to `nc`, and try exactly the same:
```
sudo su -Pl nc -s /bin/sh -c 'XDG_RUNTIME_DIR="/run/user/$(id -u)" DBUS_SESSION_BUS_ADDRESS="unix:path=${XDG_RUNTIME_DIR}/bus" export XDG_RUNTIME_DIR DBUS_SESSION_BUS_ADDRESS && bash'
$ podman network create mynetnc
mynetnc
$ podman network inspect mynetnc
[
     {
          "name": "mynetnc",
          "id": "9266bfed7a2989835396f4be4295abd2bb8b11970e31f37a951629b08f1512f1",
          "driver": "bridge",
          "network_interface": "podman1",
          "created": "2024-03-31T23:03:38.320782544+03:00",
          "subnets": [
               {
                    "subnet": "10.89.0.0/24",
                    "gateway": "10.89.0.1"
               }
          ],
          "ipv6_enabled": false,
          "internal": false,
          "dns_enabled": true,
          "ipam_options": {
               "driver": "host-local"
          }
     }
]
$ podman run --rm -it --net=mynetnc alpine ping www.google.com
Resolved "alpine" as an alias (/etc/containers/registries.conf.d/shortnames.conf)
Trying to pull docker.io/library/alpine:latest...
Getting image source signatures
Copying blob bca4290a9639 done   |
Copying config ace17d5d88 done   |
Writing manifest to image destination
Error: unable to find network with name or ID mynetnc: network not found
```

Running the latter command again with debug logs:
```
$ podman run --log-level=debug --rm -it --net=mynetnc alpine ping www.google.com
INFO[0000] podman filtering at log level debug
DEBU[0000] Called run.PersistentPreRunE(podman run --log-level=debug --rm -it --net=mynetnc alpine ping www.google.com)
DEBU[0000] Using conmon: "/usr/bin/conmon"
INFO[0000] Using sqlite as database backend
DEBU[0000] systemd-logind: Unknown object '/'.
DEBU[0000] Using graph driver overlay
DEBU[0000] Using graph root /var/lib/nc/.local/share/containers/storage
DEBU[0000] Using run root /run/user/202/containers
DEBU[0000] Using static dir /var/lib/nc/.local/share/containers/storage/libpod
DEBU[0000] Using tmp dir /run/user/202/libpod/tmp
DEBU[0000] Using volume path /var/lib/nc/.local/share/containers/storage/volumes
DEBU[0000] Using transient store: false
DEBU[0000] [graphdriver] trying provided driver "overlay"
DEBU[0000] Cached value indicated that overlay is supported
DEBU[0000] Cached value indicated that overlay is supported
DEBU[0000] Cached value indicated that metacopy is not being used
DEBU[0000] Cached value indicated that native-diff is usable
DEBU[0000] backingFs=extfs, projectQuotaSupported=false, useNativeDiff=true, usingMetacopy=false
DEBU[0000] Initializing event backend journald
DEBU[0000] Configured OCI runtime krun initialization failed: no valid executable found for OCI runtime krun: invalid argument
DEBU[0000] Configured OCI runtime crun-wasm initialization failed: no valid executable found for OCI runtime crun-wasm: invalid argument
DEBU[0000] Configured OCI runtime kata initialization failed: no valid executable found for OCI runtime kata: invalid argument
DEBU[0000] Configured OCI runtime runsc initialization failed: no valid executable found for OCI runtime runsc: invalid argument
DEBU[0000] Configured OCI runtime runc initialization failed: no valid executable found for OCI runtime runc: invalid argument
DEBU[0000] Configured OCI runtime runj initialization failed: no valid executable found for OCI runtime runj: invalid argument
DEBU[0000] Configured OCI runtime youki initialization failed: no valid executable found for OCI runtime youki: invalid argument
DEBU[0000] Configured OCI runtime ocijail initialization failed: no valid executable found for OCI runtime ocijail: invalid argument
DEBU[0000] Using OCI runtime "/usr/bin/crun"
INFO[0000] Setting parallel job count to 7
DEBU[0000] Successfully loaded network mynetnc: &{mynetnc 9266bfed7a2989835396f4be4295abd2bb8b11970e31f37a951629b08f1512f1 bridge podman1 2024-03-31 23:03:38.320782544 +0300 EEST [{{{10.89.0.0 ffffff00}} 10.89.0.1 <nil>}] [] false false true [] map[] map[] map[driver:host-local]}
DEBU[0000] Successfully loaded 2 networks
DEBU[0000] Pulling image alpine (policy: missing)
DEBU[0000] Looking up image "alpine" in local containers storage
DEBU[0000] Normalized platform linux/arm64 to {arm64 linux  [] }
DEBU[0000] Loading registries configuration "/etc/containers/registries.conf"
DEBU[0000] Loading registries configuration "/etc/containers/registries.conf.d/shortnames.conf"
DEBU[0000] Trying "docker.io/library/alpine:latest" ...
DEBU[0000] parsed reference into "[overlay@/var/lib/nc/.local/share/containers/storage+/run/user/202/containers]@ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3"
DEBU[0000] Found image "alpine" as "docker.io/library/alpine:latest" in local containers storage
DEBU[0000] Found image "alpine" as "docker.io/library/alpine:latest" in local containers storage ([overlay@/var/lib/nc/.local/share/containers/storage+/run/user/202/containers]@ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3)
DEBU[0000] exporting opaque data as blob "sha256:ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3"
DEBU[0000] Looking up image "docker.io/library/alpine:latest" in local containers storage
DEBU[0000] Normalized platform linux/arm64 to {arm64 linux  [] }
DEBU[0000] Trying "docker.io/library/alpine:latest" ...
DEBU[0000] parsed reference into "[overlay@/var/lib/nc/.local/share/containers/storage+/run/user/202/containers]@ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3"
DEBU[0000] Found image "docker.io/library/alpine:latest" as "docker.io/library/alpine:latest" in local containers storage
DEBU[0000] Found image "docker.io/library/alpine:latest" as "docker.io/library/alpine:latest" in local containers storage ([overlay@/var/lib/nc/.local/share/containers/storage+/run/user/202/containers]@ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3)
DEBU[0000] exporting opaque data as blob "sha256:ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3"
DEBU[0000] Looking up image "alpine" in local containers storage
DEBU[0000] Normalized platform linux/arm64 to {arm64 linux  [] }
DEBU[0000] Trying "docker.io/library/alpine:latest" ...
DEBU[0000] parsed reference into "[overlay@/var/lib/nc/.local/share/containers/storage+/run/user/202/containers]@ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3"
DEBU[0000] Found image "alpine" as "docker.io/library/alpine:latest" in local containers storage
DEBU[0000] Found image "alpine" as "docker.io/library/alpine:latest" in local containers storage ([overlay@/var/lib/nc/.local/share/containers/storage+/run/user/202/containers]@ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3)
DEBU[0000] exporting opaque data as blob "sha256:ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3"
DEBU[0000] Inspecting image ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3
DEBU[0000] exporting opaque data as blob "sha256:ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3"
DEBU[0000] exporting opaque data as blob "sha256:ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3"
DEBU[0000] Inspecting image ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3
DEBU[0000] Inspecting image ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3
DEBU[0000] Inspecting image ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3
DEBU[0000] using systemd mode: false
DEBU[0000] No hostname set; container's hostname will default to runtime default
DEBU[0000] Loading seccomp profile from "/usr/share/containers/seccomp.json"
DEBU[0000] Allocated lock 0 for container 8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8
DEBU[0000] parsed reference into "[overlay@/var/lib/nc/.local/share/containers/storage+/run/user/202/containers]@ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3"
DEBU[0000] exporting opaque data as blob "sha256:ace17d5d883e9ea5a21138d0608d60aa2376c68f616c55b0b7e73fba6d8556a3"
DEBU[0000] Cached value indicated that idmapped mounts for overlay are not supported
DEBU[0000] Check for idmapped mounts support
DEBU[0000] Created container "8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8"
DEBU[0000] Container "8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8" has work directory "/var/lib/nc/.local/share/containers/storage/overlay-containers/8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8/userdata"
DEBU[0000] Container "8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8" has run directory "/run/user/202/containers/overlay-containers/8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8/userdata"
DEBU[0000] Handling terminal attach
INFO[0000] Received shutdown.Stop(), terminating!        PID=2019
DEBU[0000] Enabling signal proxying
DEBU[0000] Cached value indicated that volatile is being used
DEBU[0000] Made network namespace at /run/user/202/netns/netns-024d046c-d861-eb5c-09dc-058c61934c67 for container 8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8
DEBU[0000] overlay: mount_data=lowerdir=/var/lib/nc/.local/share/containers/storage/overlay/l/HO2MFRIUYYPDA35M27FWY4M7PD,upperdir=/var/lib/nc/.local/share/containers/storage/overlay/1a3dc2e019e509594c690efdc515ea82ea72e9a9d578940292fc0de0490e69ec/diff,workdir=/var/lib/nc/.local/share/containers/storage/overlay/1a3dc2e019e509594c690efdc515ea82ea72e9a9d578940292fc0de0490e69ec/work,userxattr,volatile
DEBU[0000] Mounted container "8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8" at "/var/lib/nc/.local/share/containers/storage/overlay/1a3dc2e019e509594c690efdc515ea82ea72e9a9d578940292fc0de0490e69ec/merged"
DEBU[0000] Created root filesystem for container 8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8 at /var/lib/nc/.local/share/containers/storage/overlay/1a3dc2e019e509594c690efdc515ea82ea72e9a9d578940292fc0de0490e69ec/merged
DEBU[0000] The path of /etc/resolv.conf in the mount ns is "/etc/resolv.conf"
DEBU[0000] Successfully loaded 1 networks
DEBU[0000] Unmounted container "8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8"
DEBU[0000] Network is already cleaned up, skipping...
DEBU[0000] Cleaning up container 8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8
DEBU[0000] Network is already cleaned up, skipping...
DEBU[0000] Container 8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8 storage is already unmounted, skipping...
DEBU[0000] Removing container 8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8
DEBU[0000] Cleaning up container 8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8
DEBU[0000] Network is already cleaned up, skipping...
DEBU[0000] Container 8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8 storage is already unmounted, skipping...
DEBU[0000] Removing all exec sessions for container 8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8
DEBU[0000] Container 8b11e49f36f1320e8e91d2f79bf59c647a1183b0eebf07f0700c9fb9457e2cb8 storage is already unmounted, skipping...
DEBU[0000] ExitCode msg: "unable to find network with name or id mynetnc: network not found"
Error: unable to find network with name or ID mynetnc: network not found
DEBU[0000] Shutting down engines
```

### Describe the results you received

Using the container via the `nc` user results in `Error: unable to find network with name or ID mynetnc: network not found`.

### Describe the results you expected

Running the container via the `nc` user should succeed just like with the `yc` user.

### podman info output (yc)

```yaml
host:
  arch: arm64
  buildahVersion: 1.33.5
  cgroupControllers:
  - cpu
  - memory
  - pids
  cgroupManager: systemd
  cgroupVersion: v2
  conmon:
    package: conmon_2.1.10+ds1-1_arm64
    path: /usr/bin/conmon
    version: 'conmon version 2.1.10, commit: unknown'
  cpuUtilization:
    idlePercent: 99.42
    systemPercent: 0.3
    userPercent: 0.27
  cpus: 2
  databaseBackend: sqlite
  distribution:
    codename: trixie
    distribution: debian
    version: unknown
  eventLogger: journald
  freeLocks: 2048
  hostname: Dwight
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 201
      size: 1
    - container_id: 1
      host_id: 231072
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 201
      size: 1
    - container_id: 1
      host_id: 231072
      size: 65536
  kernel: 6.6.15-arm64
  linkmode: dynamic
  logDriver: journald
  memFree: 3398086656
  memTotal: 4110667776
  networkBackend: netavark
  networkBackendInfo:
    backend: netavark
    dns:
      package: aardvark-dns_1.4.0-5_arm64
      path: /usr/lib/podman/aardvark-dns
      version: aardvark-dns 1.4.0
    package: netavark_1.4.0-4_arm64
    path: /usr/lib/podman/netavark
    version: netavark 1.4.0
  ociRuntime:
    name: crun
    package: crun_1.14.4-1_arm64
    path: /usr/bin/crun
    version: |-
      crun version 1.14.4
      commit: a220ca661ce078f2c37b38c92e66cf66c012d9c1
      rundir: /run/user/201/crun
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +WASM:wasmedge +YAJL
  os: linux
  pasta:
    executable: ""
    package: ""
    version: ""
  remoteSocket:
    exists: false
    path: /run/user/201/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: false
  serviceIsRemote: false
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: slirp4netns_1.2.1-1_arm64
    version: |-
      slirp4netns version 1.2.1
      commit: 09e31e92fa3d2a1d3ca261adaeb012c8d75a8194
      libslirp: 4.7.0
      SLIRP_CONFIG_VERSION_MAX: 4
      libseccomp: 2.5.5
  swapFree: 0
  swapTotal: 0
  uptime: 0h 17m 41.00s
  variant: v8
plugins:
  authorization: null
  log:
  - k8s-file
  - none
  - passthrough
  - journald
  network:
  - bridge
  - macvlan
  - ipvlan
  volume:
  - local
registries: {}
store:
  configFile: /home/yc/.config/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /home/yc/.local/share/containers/storage
  graphRootAllocated: 31393230848
  graphRootUsed: 1588453376
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Supports shifting: "false"
    Supports volatile: "true"
    Using metacopy: "false"
  imageCopyTmpDir: /var/tmp
  imageStore:
    number: 1
  runRoot: /run/user/201/containers
  transientStore: false
  volumePath: /home/yc/.local/share/containers/storage/volumes
version:
  APIVersion: 4.9.3
  Built: 0
  BuiltTime: Thu Jan  1 02:00:00 1970
  GitCommit: ""
  GoVersion: go1.21.6
  Os: linux
  OsArch: linux/arm64
  Version: 4.9.3
```

### podman info output (nc)

```yaml
host:
  arch: arm64
  buildahVersion: 1.33.5
  cgroupControllers:
  - cpu
  - memory
  - pids
  cgroupManager: systemd
  cgroupVersion: v2
  conmon:
    package: conmon_2.1.10+ds1-1_arm64
    path: /usr/bin/conmon
    version: 'conmon version 2.1.10, commit: unknown'
  cpuUtilization:
    idlePercent: 99.42
    systemPercent: 0.3
    userPercent: 0.28
  cpus: 2
  databaseBackend: sqlite
  distribution:
    codename: trixie
    distribution: debian
    version: unknown
  eventLogger: journald
  freeLocks: 2048
  hostname: Dwight
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 202
      size: 1
    - container_id: 1
      host_id: 296608
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 202
      size: 1
    - container_id: 1
      host_id: 296608
      size: 65536
  kernel: 6.6.15-arm64
  linkmode: dynamic
  logDriver: journald
  memFree: 3391741952
  memTotal: 4110667776
  networkBackend: netavark
  networkBackendInfo:
    backend: netavark
    dns:
      package: aardvark-dns_1.4.0-5_arm64
      path: /usr/lib/podman/aardvark-dns
      version: aardvark-dns 1.4.0
    package: netavark_1.4.0-4_arm64
    path: /usr/lib/podman/netavark
    version: netavark 1.4.0
  ociRuntime:
    name: crun
    package: crun_1.14.4-1_arm64
    path: /usr/bin/crun
    version: |-
      crun version 1.14.4
      commit: a220ca661ce078f2c37b38c92e66cf66c012d9c1
      rundir: /run/user/202/crun
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +WASM:wasmedge +YAJL
  os: linux
  pasta:
    executable: ""
    package: ""
    version: ""
  remoteSocket:
    exists: false
    path: /run/user/202/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: false
  serviceIsRemote: false
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: slirp4netns_1.2.1-1_arm64
    version: |-
      slirp4netns version 1.2.1
      commit: 09e31e92fa3d2a1d3ca261adaeb012c8d75a8194
      libslirp: 4.7.0
      SLIRP_CONFIG_VERSION_MAX: 4
      libseccomp: 2.5.5
  swapFree: 0
  swapTotal: 0
  uptime: 0h 17m 24.00s
  variant: v8
plugins:
  authorization: null
  log:
  - k8s-file
  - none
  - passthrough
  - journald
  network:
  - bridge
  - macvlan
  - ipvlan
  volume:
  - local
registries: {}
store:
  configFile: /var/lib/nc/.config/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /var/lib/nc/.local/share/containers/storage
  graphRootAllocated: 31393230848
  graphRootUsed: 1588453376
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Supports shifting: "false"
    Supports volatile: "true"
    Using metacopy: "false"
  imageCopyTmpDir: /var/tmp
  imageStore:
    number: 1
  runRoot: /run/user/202/containers
  transientStore: false
  volumePath: /var/lib/nc/.local/share/containers/storage/volumes
version:
  APIVersion: 4.9.3
  Built: 0
  BuiltTime: Thu Jan  1 02:00:00 1970
  GitCommit: ""
  GoVersion: go1.21.6
  Os: linux
  OsArch: linux/arm64
  Version: 4.9.3
```

### Podman in a container

No

### Privileged Or Rootless

Rootless

### Upstream Latest Release

No

**Comment by 5kt** [comment]
This seems to be a duplicate of #18162 
I have replicated your exact steps and have the same issue; creating _/var/lib/cni_ on the host fixes the issue.

**Comment by Luap99** [comment]
Correct that is a dup, and in the meantime I fixed that for podman 5.0 to no longer over mount /var/lib when netavark is in use so that bug is properly fixed now.

**Comment by Gchbg** [comment]
Great catch @5kt !
